optimum_benchmark/main.py

```diff
@@ -70,7 +70,6 @@ class ExperimentConfig:
             "cache_dir": None,
             "force_download": False,
             "local_files_only": False,
-            "use_auth_token": False,
```

Comment:
This argument is deprecated in transformers

Reply:
yes was it substituted with `token` argument ?

Reply:
Not necessarily, you can use `huggingface-cli login` and it works.